ecosystem/typescript/sdk/src/plugins/coin_client.ts

```diff
@@ -59,17 +61,13 @@ export class CoinClient {
 
     // If we should create the receiver account if it doesn't exist on-chain,
     // use the `0x1::aptos_account::transfer` function.
-    const func = extraArgs?.createReceiverIfMissing ? "0x1::aptos_account::transfer" : "0x1::coin::transfer";
-
-    // If we're using the `0x1::aptos_account::transfer` function, we don't
-    // need type args.
-    const typeArgs = extraArgs?.createReceiverIfMissing ? [] : [coinTypeToTransfer];
+    const func = extraArgs?.createReceiverIfMissing ? "0x1::aptos_account::transfer_coins" : "0x1::coin::transfer";
```

Comment:
Why not just always use `transfer_coins`?  It's a minimal gas difference right?

Reply:
I really wanted to kill createReceiverIfMissing but unfortunately that'd be a breaking change (passing createReceiverIfMissing would suddenly stop doing anything). We can discuss that further separately from this change